dp/generate_climos.py

```diff
@@ -442,11 +442,14 @@ def convert_flux_var_units(input_file, climo_data):
         flux_variable = input_file.variables[flux_var]
         units = Unit.from_udunits_str(flux_variable.units)
 
-        if units in [Unit('kg / m**2 / s'), Unit('mm / s')]:
-            logger.info("Converting {} variable to units mm/day".format(flux_var))
+        if units in [Unit('kg / m**2 / s'), Unit('mm / s'), Unit('g / cm**2 / s'), Unit('cm / s')]:
```

Comment:
An option that would make this more readable is to define each set of units as a variable to use in your conditional. For example:

```python3
precip_units = {Unit('kg / m**2 / s'), Unit('mm / s'), Unit('g / cm**2 / s'), Unit('cm / s')}
snow_units = {Unit('g / m**2 / s'), Unit('cm / s')}
if units in precip_units:
    ...
```